Dockerfile

```diff
@@ -10,17 +10,20 @@ RUN apt-get update -y && \
         wget \
         unzip \
         zip \
-        libgl1 \
+        coreutils \
         qt5dxcb-plugin &&\
     rm -rf /var/lib/apt/lists/*
 
 # Define working directory
 WORKDIR /app/odv
 
 # Download and extract ODV
-RUN wget https://usegalaxy.eu/static/share/odv_5.8.0_linux-amd64.tar.gz && \
-    tar -xf odv_5.8.0_linux-amd64.tar.gz && \
-    rm odv_5.8.0_linux-amd64.tar.gz
+RUN wget https://usegalaxy.eu/static/share/odv_5.8.2_linux-amd64.tar.gz && \
+    tar -xf odv_5.8.2_linux-amd64.tar.gz && \
```

Comment:
when you use an ARG variable in Dockerfile you can template this out here I think and you only need to bump the version in one place